api4/channel.go

```diff
@@ -917,7 +917,7 @@ func viewChannel(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	view := model.ChannelViewFromJson(r.Body)
-	if view == nil || !model.IsValidId(view.ChannelId) || (view.PrevChannelId != "" && !model.IsValidId(view.PrevChannelId)) {
+	if view == nil || (view.ChannelId != "" && !model.IsValidId(view.ChannelId)) || (view.PrevChannelId != "" && !model.IsValidId(view.PrevChannelId)) {
```

Comment:
nit: 0/5 maybe break this into 2 separate if statements for clarity? Also may be worth commenting exactly why blanks are ok?

Reply:
:+1: on adding a comment for this rather complex if statement.

Reply:
Good feedback. I have broken it up and added a comment.